chromium_src/ui/views/controls/menu/menu_config.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ui/views/controls/menu/menu_config.h"
+
+#define instance instance_ChromiumImpl
+
+#include "src/ui/views/controls/menu/menu_config.cc"
+
+#undef instance
+
+namespace views {
+
+MenuConfig::MenuConfig(const MenuConfig&) = default;
+
+// static
+const MenuConfig& MenuConfig::instance() {
+  auto& config = const_cast<MenuConfig&>(instance_ChromiumImpl());
```

Comment:
hm. I missed that we'll need `const_cast` 😅. Can we do it like this?
```
const auto& config = instance_ChromiumImpl();
static const bool one_time_call = [](const MenuConfig& config) {
    auto& mutable_config = const_cast<MenuConfig&>(config);
    mutable_config.item_vertical_margin = 9;
    mutable_config.item_horizontal_padding = 16;
    return true;
  }();

return config;
```

Reply:
Done! 👍🏼